src/Dapr.PluggableComponents/Components/SecretStore/SecretStoreBulkGetRequest.cs

```diff
@@ -0,0 +1,36 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2023 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using Dapr.Proto.Components.V1;
+
+namespace Dapr.PluggableComponents.Components.SecretStore;
+
+
+/// <summary>
+/// Represents properties associated with a request to retrieve all secrets from a secret store.
+/// </summary>
+public sealed record SecretStoreBulkGetRequest
```

Comment:
Again - consistency in naming. Dapr.Client puts the verb before the modifier, so I might suggest `SecretStoreGetBulkRequest`

Reply:
The state store pluggable APIs which most closely aligns with this new secret store API uses the "Bulk" prefix first, so it's consistent within this SDK.  While I tend to prefer this naming convention, I wouldn't be opposed to using the other to match `Dapr.Client` naming conventions.  That said, it might be better to do such renaming as a separate SDK-wide change.

Reply:
I'm fine leaving it as a future PR to do any sort of broader name change. I'm not partial to either way, but I think it'd be good to be consistent across this and the other .NET SDKs eventually.

Reply:
Will consider renaming as part of a larger SDK consistency PR.